<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportDialog</class>
 <widget class="QDialog" name="ReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание отчета</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="reportSettingsGroup">
     <property name="title">
      <string>Настройки отчета</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QTextEdit" name="descriptionEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="reportTitleEdit">
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Описание:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Название работы:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dataSelectionGroup">
     <property name="title">
      <string>Выбор данных</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="includeInstrumentsCheck">
        <property name="text">
         <string>Включить информацию о приборах</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Добавить график</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="columnsLabel">
        <property name="text">
         <string>Выберите столбцы для включения в отчет:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="dataColumnsTable">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Включить</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Столбец</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Описание</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="outputSettingsGroup">
     <property name="title">
      <string>Настройки вывода</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="outputPathLabel">
        <property name="text">
         <string>Путь сохранения:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="outputPathEdit">
          <property name="placeholderText">
           <string>Выберите папку для сохранения</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Обзор</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Создать отчет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
